### DISCUSSION TITLE
是否支持硬解？

### DISCUSSION THREAD

**Discussion by moetayuko** [ywmoyue/biliuwp-lite#861] [Q&A]
我不了解 win 的视频编解码机制，但发现 biliuwp 播放视频时机器的发热量远大于 switchfin，且任务管理器里 video decode 的占用为0，因此怀疑是软解的

**Comment by ywmoyue** [comment]
@moetayuko 试试优先播放器类型设置为FFmpegInterop

**Comment by ywmoyue** [comment]
![image](https://github.com/user-attachments/assets/13ff5693-61ea-4c5b-abe1-b185739838f5)

**Comment by moetayuko** [comment]
观察发现 native 和 ffmpeg 播放视频的 cpu 占用率差不多，影视飓风这个测试片 1080p 在 7% 左右，4k 和 8k 在 12%-30% 之间，应该是都正常调用硬解了

其他弹幕少的片子，1080p biliuwp cpu 在 4% 左右，而 switchfin 仅需 2% 左右，估计发热量差异体现在这里了
